### DISCUSSION TITLE
Can we execute sql query on table saw Table?

### DISCUSSION THREAD

**Discussion by kknandagiri** [jtablesaw/tablesaw#988] [Q&A]
I was looking for a feature, if we could execute a sql query something similar to Spark sql kinda functionality with tablesaw.

**Comment by lwhite1** [comment]
Not exactly, but you can do most SQL kinds of statements in a single
statement, with the exception of Having clauses: they would take two
statements. For example, to execute something similar to a SQL LIKE with
sorted results, you could do:

```
     t.where(t.stringColumn("foo").startsWith("bar"))
         .sortOn("age");
```

There are also options for joins, and aggregation with/without grouping,
but the model for Tablesaw was data frames rather than SQL.
